code/parachain/frame/composable-traits/src/assets.rs

```diff
@@ -34,3 +35,136 @@ pub struct Asset<Balance, ForeignId> {
 	pub foreign_id: Option<ForeignId>,
 	pub existential_deposit: Balance,
 }
+
+/// Struct containing the information used to create an asset
+#[derive(Decode, Encode, Debug, Clone, PartialEq, Eq, TypeInfo)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub struct AssetInfo<Balance> {
+	/// Name of the asset.
+	pub name: Vec<u8>,
```

Comment:
should it be bounded vector? specifically why not BiBoundedVec as it was before?

Reply:
I wanted to simplify the process of constructing this structure from the FE. However, I can see value in enforcing a bounded vector from the start.
Will attempt to refactor.

Reply:
7d40673a5d8f5d5d268c3b6dc95947da8e02794b